spec/forms/idv/api_image_upload_form_spec.rb

```diff
@@ -808,6 +795,7 @@
             user_id: document_capture_session.user.uuid,
             front_image_fingerprint: an_instance_of(String),
             back_image_fingerprint: an_instance_of(String),
+            selfie_image_fingerprint: an_instance_of(String),
```

Comment:
i'm curious why this wasn't necessary before?

Reply:
The test has `expect(capture_result.failed_selfie_image_fingerprints).to be_nil`, so I'd be more surprised by a String result? It looks like this test has `liveness_checking_required` as `false`, so I wouldn't expect it to have a fingerprint for the selfie here. I've pushed up a commit to be a bit stricter in the testing of the value logged for that argument (rather than accepting any boolean).

Reply:
ok i think the confusing thing about this test is that:

- the context is `'when selfie is checked for liveness'`
- `liveness_checking_required` is false

[based on the original pr, i think that is an oversight?](https://github.com/18F/identity-idp/pull/9976/files#diff-9c26a9194a28764f16407a7d6fd1592f7a9d56c3a097f105c35bca5ce402bda5). i am going to add `let(:liveness_checking_required) { true} ` to the spec

the reason that the `capture_result.failed_selfie_image_fingerprint` should be `nil` is because in this spec, the selfie_status is `:success`. `failed_selfie_image_fingerprint` is [only set if the selfie_status is](https://github.com/18F/identity-idp/blob/main/app/models/document_capture_session.rb#L58) `:fail:`

Reply:
updated in https://github.com/18F/identity-idp/pull/12197/commits/7c53d2bb07f3e4797c6691be6b8a205c5910e1b6